### DISCUSSION TITLE
Connect next.js with drizzle ORM to AWS RDS postgres database

### DISCUSSION THREAD

**Discussion by BryanGabrielAtangana** [community/community#139034] [Programming Help]
### Body

Step by step, how can I connect my AWS RDS database db to my drizzle ORM in my next.js app to be able to make migrations.
I have created my db on AWS RDS and also lunched my EC2 instance. I'm blocked on the connection with my local app.

Thanks for your help 😇

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by sabbirsami** [comment]
# Connecting AWS RDS to Drizzle ORM in Next.js and Running Migrations

## Problem Overview

I have set up my AWS RDS database and launched my EC2 instance. Now, I need help connecting my AWS RDS database to my Next.js app using Drizzle ORM so that I can run database migrations.

I have created the following so far:
- AWS RDS PostgreSQL/MySQL database.
- EC2 instance running and configured.
- A Next.js app.

However, I am stuck on how to properly connect the AWS RDS database to Drizzle ORM and how to run migrations from my local Next.js app.

---

## Step-by-Step Solution

### 1. Install Drizzle ORM in Your Next.js App

First, ensure Drizzle ORM is installed in your Next.js app along with the required packages for your database type (PostgreSQL or MySQL):

```bash
npm install drizzle-orm drizzle-orm-postgres drizzle-kit pg
